### PR TITLE
Update user-credentials.adoc related to secret

### PR DESCRIPTION
**Pull request by quanchentg**

### CODE REVIEW

File: modules/user-access/pages/user-credentials.adoc [hunk 1]
```diff
@@ -72,33 +72,40 @@ These operations include:
 
 == Secrets
 
-Secrets are unique strings that serve as a user's credentials in certain circumstances. A user can have multiple secret strings. Each secret is associated with one user and their role for one graph. If the role is revoked, the secret also becomes invalid.
+Secrets are unique strings that serve as a user’s credentials in certain circumstances. A user can have multiple secret strings. Each secret binds to the user who creates it only and inherits all permissions and privileges from the owner. Starting from 4.1.0, secret is no longer bound to graphs and tokens.
```
Comment (arun-TG-PM) [inline modules/user-access/pages/user-credentials.adoc]: Add a separate line as "Note " for : As of version 4.1.0, secrets are no longer bound to specific graphs and tokens.

File: modules/user-access/pages/user-credentials.adoc [hunk 2]
```diff
@@ -72,33 +72,40 @@ These operations include:
...
-
-At least one of the database access privileges on the given graph: `READ_DATA`, `CREATE_DATA`, `UPDATE_DATA`, `DELETE_DATA` or `EXECUTE_LOADINGJOB`. The built-in `queryreader` role satisfies this requirement.
+Use GSQL Rest API:
+[source,gsql]
+----
+POST /gsql/v1/secrets?userName=<user>&value=<secret content>&alias=<alias>
+----
+Use the `POST method to call secrets API` to create secrets.
 
+* Case 1: `userName` is optional, if `userName` is not specified, it creates secret for default user who logins.
```
Comment (arun-TG-PM) [inline modules/user-access/pages/user-credentials.adoc]: Let's make it more clear.
"Case 1: The userName parameter is optional. If not specified, the secret is created for ......"

File: modules/user-access/pages/user-credentials.adoc [hunk 3]
```diff
@@ -72,33 +72,40 @@ These operations include:
...
+* Case 2: `alias` is optional, if `alias` is not specified, randomly generated aliases begin with AUTO_GENERATED_ALIAS_ and include a random 7-character string.
```
Comment (arun-TG-PM) [inline modules/user-access/pages/user-credentials.adoc]: Same as above, split the sentence for more clarity

File: modules/user-access/pages/user-credentials.adoc [hunk 4]
```diff
@@ -72,33 +72,40 @@ These operations include:
...
+* Case 3: `value` is optional, if `value` is not specified, randomly generated 32-character string.
```
Comment (arun-TG-PM) [inline modules/user-access/pages/user-credentials.adoc]: Same as above.
"Case 3: The value parameter is optional. If not specified, a random 32-character string will be generated."

File: modules/user-access/pages/user-credentials.adoc [hunk 5]
```diff
@@ -72,33 +72,40 @@ These operations include:
...
 
+==== Required privilege
+There is no privilege needed if user creates secrets for self.
```
Comment (arun-TG-PM) [inline modules/user-access/pages/user-credentials.adoc]: For more clarity: No privilege is required to create secrets for oneself. The superuser role is needed to create secrets for other users.

Comment (arun-TG-PM) [reply modules/user-access/pages/user-credentials.adoc]: Use the same for the below for other reptitions

**Review by arun-TG-PM — COMMENTED** [review]
Great work! Please review my comments and resolve them ASAP.

File: modules/user-access/pages/user-credentials.adoc (hunk 5 above)
Comment (zhihuapengtg) [reply modules/user-access/pages/user-credentials.adoc]: @arun-TG-PM   We have addressed your comments, please help to review it again, thanks!

**Review by arun-TG-PM — APPROVED** [review]

**Comment by arun-TG-PM** [comment]
@zhihuapengtg @dadongwang-tg - Do you have DOC JIRA ticket for this?

**Comment by zhihuapengtg** [comment]
> @zhihuapengtg @dadongwang-tg - Do you have DOC JIRA ticket for this?

hello arun, I think should be this ticket: https://graphsql.atlassian.net/browse/DOC-2227

**Pull request merged**